ledger/src/blockstore.rs

```diff
@@ -71,6 +71,21 @@ pub mod blockstore_purge;
 
 pub const BLOCKSTORE_DIRECTORY: &str = "rocksdb";
 pub const ROCKSDB_TOTAL_SST_FILES_SIZE: &str = "rocksdb.total-sst-files-size";
+pub const ROCKSDB_SIZE_ALL_MEM_TABLES: &str = "rocksdb.size-all-mem-tables";
+pub const ROCKSDB_NUM_SNAPSHOTS: &str = "rocksdb.num-snapshots";
+pub const ROCKSDB_OLDEST_SNAPSHOT_TIME: &str = "rocksdb.oldest-snapshot-time";
+pub const ROCKSDB_ACTUAL_DELAYED_WRITE_RATE: &str = "rocksdb.actual-delayed-write-rate";
+pub const ROCKSDB_IS_WRITE_STOPPED: &str = "rocksdb.is-write-stopped";
+pub const ROCKSDB_BLOCK_CACHE_CAPACITY: &str = "rocksdb.block-cache-capacity";
+pub const ROCKSDB_BLOCK_CACHE_USAGE: &str = "rocksdb.block-cache-usage";
+pub const ROCKSDB_BLOCK_CACHE_PINNED_USAGE: &str = "rocksdb.block-cache-pinned-usage";
+pub const ROCKSDB_ESTIMATE_TABLE_READERS_MEM: &str = "rocksdb.estimate-table-readers-mem";
+pub const ROCKSDB_MEM_TABLE_FLUSH_PENDING: &str = "rocksdb.mem-table-flush-pending";
+pub const ROCKSDB_COMPACTION_PENDING: &str = "rocksdb.compaction-pending";
+pub const ROCKSDB_NUM_RUNNING_COMPACTIONS: &str = "rocksdb.num-running-compactions";
+pub const ROCKSDB_NUM_RUNNING_FLUSHES: &str = "rocksdb.num-running-flushes";
+pub const ROCKSDB_ESTIMATE_OLDEST_KEY_TIME: &str = "rocksdb.estimate-oldest-key-time";
+pub const ROCKSDB_BACKGROUND_ERRORS: &str = "rocksdb.background-errors";
```

Comment:
All of these are now available in the rocksdb crate; see https://github.com/solana-labs/solana/pull/23147 where I updated the constant we already had in our code